source/features/navigate-pages-with-arrow-keys.tsx

```diff
@@ -2,13 +2,25 @@ import select from 'select-dom';
 
 import features from '.';
 
+const nextPageButtonSelectors = [
+	'a.next_page', // Issue/PR list, Search
+	'.paginate-container > .BtnGroup .btn:last-child', // Commits
+	'.paginate-container > .pagination > :last-child' // Releases
```

Comment:
Should `.btn` be dropped from both?
```suggestion
	'.paginate-container > .BtnGroup > :last-child', // Commits
	'.paginate-container > .pagination > :last-child' // Releases
```